hack/make/binary

```diff
@@ -36,7 +36,7 @@ if [ "$(go env GOOS)" == "linux" ] ; then
 	esac
 fi
 
-if [ "$IAMSTATIC" == "true" ] && [ "$(go env GOHOSTOS)" == "linux" ] && [ "$DOCKER_EXPERIMENTAL" ]; then
+if [ "$IAMSTATIC" == "true" ] && [ "$(go env GOHOSTOS)" == "linux" ]; then
```

Comment:
No no don't worry about it do what u think is best

On Tuesday, March 15, 2016, Riyaz Faizullabhoy <notifications@github.com>
wrote:

> In hack/make/binary
> <https://github.com/docker/docker/pull/21003#discussion_r56258998>:
>
> > @@ -36,7 +36,7 @@ if [ "$(go env GOOS)" == "linux" ] ; then
> >  	esac
> >  fi
> >
> > -if [ "$IAMSTATIC" == "true" ] && [ "$(go env GOHOSTOS)" == "linux" ] && [ "$DOCKER_EXPERIMENTAL" ]; then
> > +if [ "$IAMSTATIC" == "true" ] && [ "$(go env GOHOSTOS)" == "linux" ]; then
>
> Edit: I tried this and it seems like it just complicates readability
> because of the complexity between $IAMSTATIC and GOHOSTOS vs. GOOS. I can
> refactor if you'd prefer into the switch statement but I actually think the
> current separation might be preferred.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/21003/files/2c3e9e57949d23b4453b21339da56f0424ecbe42#r56258998>
>


-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>